docs/sqllab.rst

```diff
@@ -123,6 +123,29 @@ database configuration:
 Here, "version" should be the version of your Presto cluster. Support for this
 functionality was introduced in Presto 0.319.
 
+You also need to enable the feature flag in your `superset_config.py`, and you
+can optionally specify a custom formatter. Eg:
+
+.. code-block:: python
+
+	def presto_query_cost_formatter(result):
```

Comment:
could we add a type to this so that people know what valid params there are? Or add that in a comment somewhere?

Reply:
Done! Sorry, I'm still not fully used to types in Python, and I keep forgetting them. :-P